packages/server-wallet/src/wallet/__test__/add-signed-state.test.ts

```diff
@@ -22,14 +19,32 @@ describe('addSignedState', () => {
 
   const BOB_SIGNATURE =
     '0x36a5fd36a1c9a85afdeee3f9471579656eefceb08bc0ff53d194a67d6433c6385cc8c9aa049306fc7cce901f7b3345bccde311cceadc74a40a89a9d74d86d9b91b';
-  it('throws on an invalid signature', async () => {
-    const signedState = addHash({
-      ...createState(),
-      signatures: [{signer: alice().address, signature: BOB_SIGNATURE}],
-    });
-
-    await expect(store.addSignedState(undefined, signedState, knex as any)).rejects.toThrow(
-      'Invalid signature'
-    );
+
+  const channelId = '0x00';
+  it('throws when the signer is not a participant', async () => {
+    const signedState: WireSignedState = {
+      appData: '0x',
+      appDefinition: '0x0000000000000000000000000000000000000000',
+      isFinal: false,
+      turnNum: 0,
+      outcome: [
+        {
+          assetHolderAddress: '0x2222E21c8019b14dA16235319D34b5Dd83E644A9',
+          allocationItems: [
+            {destination: '0x2222E21c8019b14dA16235319D34b5Dd83E644A9', amount: '0x00'},
+          ],
+        },
+      ],
+      participants: [],
+      channelNonce: 1,
+      channelId,
+      chainId: '0x01',
+      challengeDuration: 9001,
+      signatures: [BOB_SIGNATURE],
+    };
```

Comment:
Notes: 

- this test asserts something a bit different now, since the behaviour has changed
- this mock is a bit ugly.